src/main/java/org/broadinstitute/hellbender/tools/spark/sv/SVIntervalTree.java

```diff
@@ -0,0 +1,904 @@
+package org.broadinstitute.hellbender.tools.spark.sv;
+
```

Comment:
It's unfortunate it was done this way to start.  Auditing the code for off by one errors if we converted was deemed impossible at this stage of things.